app/services/submit_application.rb

```diff
@@ -4,6 +4,13 @@ def initialize(url, token)
     @token = token
   end
 
+  def available?
```

Comment:
Not a blocker for merging, but could we use this before starting/submitting in the future?